Simulation/6DOF_RK4/dynamics/rocket.py

```diff
@@ -127,7 +226,86 @@ def I_inv(self, total_mass=rocket_total_mass):
                                        1/((total_mass/12) * (self.l**2 + 3*self.r_r**2)),
                                        1/((total_mass/12) * (self.l**2 + 3*self.r_r**2))])
     
+    # Appends data to rocket's storage
```

Comment:
there has to be a better way of doing this (but we shouldnt worry about that right now)

Reply:
I think the solution is pass the dictionary to those functions and have them append the appropriate data to each list

Reply:
bc this is kinda rly ugly rn

Reply:
It is kinda ugly, but I think it's better to have it centralized rather than spreading the data collection out over a bunch of functions.